src/fw-update/fw-update-device.cpp

```diff
@@ -140,7 +140,10 @@ namespace librealsense
     {
         // checking fw compatibility (covering the case of recovery device with wrong product line fw )
         std::vector<uint8_t> buffer((uint8_t*)fw_image, (uint8_t*)fw_image + fw_image_size);
-        if (!check_fw_compatibility(buffer))
+
+        // TODO: DFU issue - remove d500_device usage when HKR will support D5XX_RECOVERY device
+        bool d500_device = true;
```

Comment:
Here we cannot know if it's a D500 device right?

Reply:
Yes, because we get here with D4XX_RECOVERY device.
If HKR will support FIRMWARE_UPDATE_ID, then we can compare to it.
Or even better, if in the future, HKR will support new D5XX_RECOVERY device.